static/includes/apps/AppInstallWizardTemporaryAndTmpfsDirectories.md

```diff
@@ -0,0 +1,15 @@
+&NewLine;
+
+{{< expand "Setting Up Temporary Directory Volumes" "v" >}}
+To allow TrueNAS to create temporary directories as storage volumes instead of using datasets, you have two options: set **Type** set to **Temporary (Temporary directory created on the disk)** or **tmpfs (Temporary directory created on the RAM)**.
+This adds a directory on the disk for the application logs or transcode data in the ???? **ix-apps** dataset, located on the pool selected as the apps pool.
```

Comment:
Does this need further investigation or is the content good?
```suggestion
This adds a directory on the disk for the application logs or transcode data in the **ix-apps** dataset, located on the pool selected as the apps pool.
```

Reply:
This is the info Stavros provided, but I think I'll clarify it a bit more.

Reply:
oops.... just saw my ???? in that sentence.